xcache/xrootd/50-docker-paths.cfg

```diff
@@ -0,0 +1,11 @@
+# Make the caching plugin aware of the oss.space locations
+if nameed stash-origin
+  # do nothing
+else if named stash-origin-auth
+  # do nothing
+else
+  pfc.spaces data meta
```

Comment:
I am not sure how this is a good idea. I have stash caches that uses multiple disks and meta.

Reply:
Doesn't this just make the caching plugin aware of the data and meta `oss.space` locations?